docs/promtail.md

```diff
@@ -21,7 +21,8 @@ The term "label" here is used in more than one different way and they can be eas
   will have a label \_\_meta_kubernetes_pod_label_name with value set to "foobar".
 * There are other \_\_meta_kubernetes_* labels based on the Kubernetes metadadata, such as the namespace the pod is
   running (\_\_meta_kubernetes_namespace) or the name of the container inside the pod (\_\_meta_kubernetes_pod_container_name)
-* The label \_\_path\_\_ is a special label which Promtail will read to find out where the log files are to be read in.
+* The label \_\_path\_\_ is a special label which Promtail will read to find out where the log files are to be read in,
+  it will be translated into a label named `filename` to ensure uniqueness of the streams.
```

Comment:
This is not strictly accurate.  `__path__` is a glob-syntax string promtail uses to find files to tail.  The actual filename of the file tails is emitted in `filename`.

@slim-bean is `filename` the pre or post symlink name?

Reply:
@tomwilkie filename will be pre-symlink, the symlink expansion was moved inside the tailer and that doesn't leave the tailer anymore